### PR TITLE
improvement: Unsubscribe tokens

### PR DESCRIPTION
**Pull request by skanderm**
Currently generates an unsubscribe link for emails, but the same code can be used to create unsubscribe links for any subscription. Included is the scaffolding for managing all subscriptions, but more work is needed for creating that management UI.

Includes a simple unsubscribe success page, which will need some UI/UX work.

image
image

### CODE REVIEW

**Review by paulcretu — APPROVED** [review]

File: server/lib/orcasite/notifications/resources/subscription.ex [hunk 1]
```diff
@@ -19,22 +20,32 @@ defmodule Orcasite.Notifications.Subscription do
 
   actions do
     defaults [:create, :read, :update, :destroy]
+    defaults [:create, :read, :update, :destroy]
```
Comment (paulcretu) [inline server/lib/orcasite/notifications/resources/subscription.ex:23]: things are duplicated?

File: server/lib/orcasite/notifications/resources/subscription.ex [hunk 2]
```diff
@@ -3,9 +3,10 @@ defmodule Orcasite.Notifications.Subscription do
     extensions: [AshAdmin.Resource, AshAuthentication],
     data_layer: AshPostgres.DataLayer
 
-  alias Orcasite.Notifications.{Event, Notification, NotificationInstance, Subscriber}
+  alias Orcasite.Notifications.{Event, Notification, SubscriptionNotification, Subscriber}
```
Comment (paulcretu) [inline server/lib/orcasite/notifications/resources/subscription.ex:6]: seems wrong?

Comment (skanderm) [reply server/lib/orcasite/notifications/resources/subscription.ex]: Good catch

File: server/lib/orcasite/notifications/resources/subscription.ex (hunk 1 above)
Comment (skanderm) [reply server/lib/orcasite/notifications/resources/subscription.ex]: Yep, bad merge. Fixing.

**Review by paulcretu — APPROVED** [review]

**Pull request merged**